### ISSUE TITLE
google knowledge panel

### ISSUE THREAD

**Issue by xhdix** [ooni/ooni.org#509]
Contact Google about it:
https://support.google.com/knowledgepanel/contact/posts_on_google
![image](https://user-images.githubusercontent.com/12384263/82848287-9448cb80-9efb-11ea-9976-f9be4e840db3.png)

**Comment by hellais** [comment]
Yeah I guess it's worth a shot, though I wonder what happens if there is a naming conflict like in this case.

**Comment by xhdix** [comment]
Will probably look like this:
![image](https://user-images.githubusercontent.com/12384263/82892113-917bc400-9f57-11ea-9472-8e2f399673ae.png)

**Comment by xhdix** [comment]
:D 
![image](https://user-images.githubusercontent.com/12384263/91992921-64506600-ed3d-11ea-8a56-a6fb37693d47.png)
